src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -59,7 +59,7 @@ public <T> Future<T> post(String endpoint, T entity, Class<T> responseType, Requ
       .sendJson(entity)
       .map(HttpResponse::bodyAsJsonObject)
       .map(body -> body.mapTo(responseType))
-      .onFailure(log::error);
+      .onFailure(t -> log.error("Error while post request", t));
```

Comment:
why the new approach is better that previous one, because it was not clear that post endpoint is invoked?

Reply:
https://github.com/folio-org/mod-finance/pull/230/files/1642ee63a1dbc85e11514992c8fa6ae6893e0c31#diff-f695200a0953f109601e796e8d77f7e2bb1eb261935eb7aebb76b3045c7eeb1aR51
it is post method, that is why post endpoint is being invoked. if it is not useful that previous one, I will revert this

Reply:
Reverted